azure-pipelines.yml

```diff
@@ -23,7 +23,7 @@ pool:
 
 variables:
   MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
-  MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+  MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER) -Dcheckstyle.skip=true -Drat.skip=true'
```

Comment:
You mean we do not check the code style since now?

Reply:
@yanghua Meant to let the GitHub Action CI jobs check this. Azure CI does the heavy lifting for actual tests.